### ISSUE TITLE
No such file or directory error  for C:\Program after starting training when building custom classifier

### ISSUE THREAD

**Issue by SimonKravis** [PetervanLunteren/EcoAssist#28]
No such file or directory error  for C:\Program in Session log after starting training on Windows box with GPU when building custom classifier as shown below, but processing continues anyway.

Also message about out-of-date YOLOv5

>Starting training process...

[34m[1mtrain: [0mweights=C:\Program Files\EcoAssist_files\pretrained_models\md_v5a.0.0.pt, cfg=, data=D:\Goanna Cam pics 2022_23 Ains_Maj 1.8M files 6.30Tb\Fox Magpie Chough 2\dataset.yaml, hyp=..\..\..\..\..\..\Program Files\EcoAssist_files\yolov5\data\hyps\hyp.scratch-low.yaml, epochs=300, batch_size=-1, imgsz=1280, rect=False, resume=False, nosave=False, noval=False, noautoanchor=False, noplots=False, evolve=None, bucket=, cache=None, image_weights=False, device=0, multi_scale=False, single_cls=False, optimizer=SGD, sync_bn=False, workers=4, project=Training 1000, name=exp, exist_ok=False, quad=False, cos_lr=False, label_smoothing=0.0, patience=100, freeze=[12], save_period=-1, seed=0, local_rank=-1, entity=None, upload_dataset=False, bbox_interval=-1, artifact_alias=latest

[34m[1mgithub: [0m YOLOv5 is out of date by 107 commits. Use 'git pull' or 'git clone https://github.com/ultralytics/yolov5' to update.

fatal: cannot change to 'C:\Program': No such file or directory

YOLOv5  2023-10-18 Python-3.8.15 torch-1.10.1 CUDA:0 (NVIDIA GeForce RTX 3060, 8192MiB)

**Comment by PetervanLunteren** [comment]
This seems to be a problem regarding a space in the file path, more specifically `C:\Program Files\EcoAssist_files\`. I can't recreate the error on my M1 mac or Windows 11. Can you copy the model file `C:\Program Files\EcoAssist_files\pretrained_models\md_v5a.0.0.pt` and the YAML file `C:\Program Files\EcoAssist_files\yolov5\data\hyps\hyp.scratch-low.yaml` to a directory without spaces and try to run training again? You can select the model via `custom model` under "Retrain from" and the YAML file via `Other` under "Hyperparameter configuration file".
